### PR TITLE
Swift 3 (XCode 8 Beta 3)

### LINKED ISSUE #144: Swift 3 compatibility
Hello, thanks for your amazing work.

Do you know when it will be Swift 3 ready?

If you create a "swift3" branch, maybe I can take some time to submit a PR.

### PR DESCRIPTION
**Pull request by Tulleb**
Swift 3 compatibility.
Maybe it would be better to merge it into another branch, but I can't create it.

Closes #144.

---
This change is [Reviewable](https://reviewable.io/reviews/mobileplayer/mobileplayer-ios/145)

### CODE REVIEW

File: MobilePlayer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -661,16 +661,23 @@
 					};
 					5F56CB801C040E8D00EF50C4 = {
 						CreatedOnToolsVersion = 7.1.1;
+						DevelopmentTeam = 9E43V6CC9L;
+						DevelopmentTeamName = "Guillaume Bellut";
```
Comment (isair) [inline MobilePlayer.xcodeproj/project.pbxproj]: `DevelopmentTeam` and `DevelopmentTeamName` keys should be removed.

Comment (Tulleb) [inline MobilePlayer.xcodeproj/project.pbxproj]: My bad, I thought I did it already.

**Comment by marciogranzotto** [comment]
what's the status on this? Now that xcode 8 and iOS 10 are out of beta, that's kind of required

**Comment by Tulleb** [comment]
I don't know @marciogranzotto. It's been over a month I'm waiting for a merge now.
I decided to drop this framework and to migrate to [this one instead](https://github.com/AlexLittlejohn/ALCameraViewController).

This PR was for XCode 8 Beta 3 by the way. I'm almost sure it won't work with the last released version.

**Comment by marciogranzotto** [comment]
Yep, I've forked your swift3 branch so I can update it to the release version. I'll make a PR later today

**Comment by isair** [comment]
Closing as code in master branch is now Swift 3 compatible: https://github.com/mobileplayer/mobileplayer-ios/commit/b728d0481d44af1101ae14b97e156230f9b7f470

**Pull request closed without merging**